files/es/web/api/network_information_api/index.md

```diff
@@ -15,7 +15,7 @@ La Network Information (Información de red) API provee información sobre el si
 
 ```js
 var conexion = navigator.connection || navigator.mozConnection || navigator.webkitConnection;
-var tipo = connection.effectiveType;
+var tipo = conexion.effectiveType;
```

Comment:
@roberthgnz Gracias por contribuir!

probablemente podría ser un cambio temporal dado que el [código en ingles](https://github.com/mdn/content/blob/main/files/en-us/web/api/network_information_api/index.md?plain=1) ha cambiado y también no traducimos el código a español. 
@Graywolf9, crees que este bien aceptar?

Reply:
Muchas gracias por el PR @roberthgnz !

Veo que a lo largo del documento se sigue usando como `connection`, así mismo como comenta Juan, solo traducimos comentarios y cadenas de texto que no alteren el funcionamiento del ejemplo. Creo que la corrección sería más bien cambiar el `conexion`de la linea 17 por `connection` como en el resto del documento

Reply:
mucho ojo,  hay otras lineas abajo que usan `conexion`, creo linea 22 y 24

Reply:
Tienes razón y en el ejemplo siguiente vuelve a `connection`

Reply:
Hola @Graywolf9 al parecer se ha mal entendido el cambio. Originalmente estaba 

```
var conexion = navigator.connection || navigator.mozConnection || navigator.webkitConnection;
var tipo = connection.effectiveType;
```

Al ejecutar el código de prueba da el error: 

 **Uncaught ReferenceError: connection is not defined**

Al ver todo lo expuesto anteriormente, en realidad el cambio debe ser a la variable conexion por connection y no `var tipo = conexion.effectiveType;` tal como propuse.